sdk/daml-lf/interpreter/src/test/scala/com/daml/lf/speedy/UpgradeTest.scala

```diff
@@ -40,19 +34,20 @@ class UpgradeTest(majorLanguageVersion: LanguageMajorVersion)
   private[this] implicit def parserParameters(implicit
       pkgId: Ref.PackageId
   ): ParserParameters[this.type] =
-    ParserParameters(pkgId, languageVersion = majorLanguageVersion.dev)
+    ParserParameters(pkgId, languageVersion = LanguageMajorVersion.V2.dev)
 
   val pkgId1 = Ref.PackageId.assertFromString("-pkg1-")
   private lazy val pkg1 = {
     implicit def pkgId: Ref.PackageId = pkgId1
     p""" metadata ( '-upgrade-test-' : '1.0.0' )
     module M {
 
-      record @serializable T = { sig: Party, aNumber: Int64 };
+      record @serializable T = { sig: Party, obs: Party, aNumber: Int64 };
       template (this: T) = {
         precondition True;
         signatories M:sigs (M:T {sig} this) (None @Party);
-        observers Nil @Party;
+        observers '-pkg1-':M:sigs (M:T {obs} this) (None @Party);
```

Comment:
I was a bit confused as to why `observers` was using the `sigs` function at first. Maybe you could rename it to `mkList` or something like that for clarity?

Reply:
ok